src/Attachment.tsx

```diff
@@ -13,6 +15,50 @@ const buttons = (
         { buttons.map((button, index) => <li key={ index }><button onClick={ () => onClickButton(button.type, button.value) }>{ button.title }</button></li>) }
     </ul>;
 
+const isYoutube = (src: string) => {
+    let youtubeUrl = new URI(src);
```

Comment:
Our convention in WebChat is to use `const` when possible. Please humor me here and below.